ceph.spec.in

```diff
@@ -830,8 +830,8 @@ if [ $1 = 0 ] ; then
   SERVICE_LIST=$(systemctl | grep -E '^ceph-mon@|^ceph-create-keys@|^ceph-osd@|^ceph-mds@|^ceph-disk-'  | cut -d' ' -f1)
   if [ -n "$SERVICE_LIST" ]; then
     for SERVICE in $SERVICE_LIST; do
-      /usr/bin/systemctl --no-reload disable $SERVICE > /dev/null 2>&1 || :
-      /usr/bin/systemctl stop $SERVICE > /dev/null 2>&1 || :
+      %{_bindir}/systemctl --no-reload disable $SERVICE > /dev/null 2>&1 || :
+      %{_bindir}/systemctl stop $SERVICE > /dev/null 2>&1 || :
```

Comment:
Here (and in several other chunks), I'm not completely sure about this. People usually use the hard-coded paths in post scripts although I'm not sure what is the reason for that.